### ISSUE TITLE
New screen savers - can you add support?

### ISSUE THREAD

**Issue by Niiix** [JohnCoates/Aerial#279]
http://www.macrumors.com/2016/12/16/apple-tv-new-aerial-screen-savers/

Thanks!

**Comment by alistairmcmillan** [comment]
John's screensaver should pick up the new aerial movies automatically. If they don't please check you're on the latest version and make sure the screen saver isn't being blocked by a firewall or proxy.

Duplicate of #276.
